.github/workflows/pullrequest.yml

```
@@ -403,28 +133,10 @@ jobs:
-          yarn install --immutable
-
-      - name: Restore generated files
-        uses: runs-on/cache/restore@v4
-        id: restore-generated-files-cache
+      - name: load-deps
+        uses: ./.github/actions/load-deps
         with:
-          path: |
-            ${{ env.GENERATED_FILES }}
-          key: generated-files-${{ github.sha }}
+          skip-checkout: true
```

actionlint (1.7.4)

input "skip-checkout" is not defined in action "Load Dependencies" defined at "./.github/actions/load-deps". available inputs are "generated_files", "skip_checkout"

(action)